presto-raptor/src/main/java/com/facebook/presto/raptor/storage/organization/ShardCompactionManager.java

```diff
@@ -179,28 +169,38 @@ private void discoverShards()
             if (!metadataDao.isCompactionEnabled(tableId)) {
                 continue;
             }
-
-            Table table = metadataDao.getTableInformation(tableId);
-
-            CompactionSetCreator compactionSetCreator = getCompactionSetCreator(table);
-            Set<ShardMetadata> shards = getFilteredShards(entry.getValue(), table);
-
-            addToCompactionQueue(compactionSetCreator, tableId, shards);
+            Collection<OrganizationSet> compactionSets = createCompactionSets(tableId, entry.getValue());
+            compactionSets.stream()
```

Comment:
Why? 
I actually like having this variable. It is only an additional line of code and it is a lot easier to debug and read. 
When I had it as a single line it was easier to miss the enqueue call.